examples/dot_product/README.md

```diff
@@ -0,0 +1,45 @@
+# Dot product
+
+This example contains the oneDPL-based implementation of dot product based on `std::transform_reduce`.
+
+| Optimized for                   | Description                                                                                    |
+|---------------------------------|------------------------------------------------------------------------------------------------|
+| OS                              | Linux* Ubuntu* 18.04                                                                           |
```

Comment:
I see that some examples support Windows OS, some - not. 
Is that expected? Or are we going to change it in future?